formapp/templates/controlpanel.html

```diff
@@ -378,14 +400,23 @@ <h2>Clear data</h2>
       document.getElementById("studentNum").value = Object.keys(res).length;
     }
   }
-
+  
   updateStudents();
   setInterval(updateStudents, 1000);
-
-  function changeMode() {
+  
+  let pollButton = document.getElementById("pollButton");
+  if (pollButton.value = "Start new poll"){
+    pollButton.onclick = startPoll
+  } else if (pollButton.value = "End poll"){
+    pollButton.onclick = endPoll
+  }
```

Comment:
Was this just moved?

Reply:
I realized I didn't need that in the most recent commit